src/relay/transforms/device_planner.cc

```diff
@@ -765,7 +765,8 @@ class DeviceCapturer : public ExprMutator {
 
   IRModule Capture() {
     VLOG_CONTEXT << "CaptureDevices";
-    IRModule result(/*functions=*/{}, mod_->type_definitions, mod_->Imports(), mod_->source_map);
+    IRModule result(/*functions=*/{}, mod_->type_definitions, mod_->Imports(), mod_->source_map,
```

Comment:
Hmm, maybe we should add a WithFields for IRModule.

Reply:
Yes! Thanks for reminding me, filed CORE-126.